test/mysql_handler/mysql_test.go

```diff
@@ -90,104 +110,81 @@ func TestMySQLHandler(t *testing.T) {
 
 	Convey("Connect over TCP", t, func() {
 
-		Convey("Without SSL", func() {
+		// Geri suggests: No TLS Upstream, TLS Downstream and sslmode default
+		//
+		Convey("No TLS Upstream, TLS Downstream and sslmode default", func() {
```

Comment:
This confused me, because "upstream" generally refers to the server, which in this case would be mysql.  It's still not clear to me what's going on.